feature-libs/product-configurator/rulebased/components/config/configurator-ui-config.ts

```diff
@@ -0,0 +1,13 @@
+export abstract class ConfiguratorUIConfig {
+  rulebasedConfigurator: {
+    quantityDebounceTime?: number;
+    inputDebounceTime?: number;
+  };
+}
+
+export const DefaultConfiguratorUIConfig: ConfiguratorUIConfig = {
+  rulebasedConfigurator: {
+    quantityDebounceTime: 750,
```

Comment:
Is there a reason why the debounce time was increased for the quantity from 500 to 750?

Reply:
I tested a lot and had the impression that 500ms are to short. Especially when clicking the plus or minus sign with the mouse.
The input field is only about typing no mouse interaction. 500ms seems to be sufficient.

Reply:
ok